lib/user.rb

```diff
@@ -0,0 +1,35 @@
+require 'bcrypt'
+
+class User 
+  attr_reader :id, :name, :username, :email
+
+  def initialize(id: , name: , username: , email: )
+    @id = id
+    @name = name
+    @username = username
+    @email = email
+  end
+
+  def self.create(name: , username: , email: , password: )
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+    else
+      connection = PG.connect(dbname: 'chitter')
+    end
+    encrypted_password = BCrypt::Password.create(password)
+    user = connection.exec("INSERT INTO users (user_name, username, email, user_password) VALUES('#{name}', '#{username}', '#{email}', '#{encrypted_password}') RETURNING id, user_name, username, email;")
```

Comment:
I'd separate the sql to keep line length down and clarity up (i have not done this myself tho but intend on doing do)